include/constants/battle.h

```diff
@@ -186,7 +186,7 @@
 #define HITMARKER_x4000000              (1 << 26)
 #define HITMARKER_CHARGING              (1 << 27)
 #define HITMARKER_FAINTED(battler)      (gBitTable[battler] << 28)
-#define HITMARKER_FAINTED2(battler)     (1 << (28 + battler))
+#define HITMARKER_FAINTED2(battler)     ((1 << 28) << battler) //(1 << (28 + battler))
```

Comment:
Evaluate `1 << 28` as an integer literal i.e. `0x10000000`

Reply:
Why? None of the other macros do, and bit shifts are easier to read